waltz-ng/client/physical-flows/physical-flow-registration.js

```diff
@@ -72,6 +75,17 @@ function validate(specification, flowAttributes, targetEntity) {
 }
 
 
+function getSimilarFlows(specification, flowAttributes, targetEntity, existingFlows = []) {
```

Comment:
Whats the purpose of passing in `flowAttributes` and `targetEntity` - they don't seem to be used other than in the `if`